src/js/helpers/bulkDownloadHelper.js

```diff
@@ -60,3 +60,8 @@ export const requestArchiveFiles = (params) => apiRequest({
 export const requestDictionaryContent = () => apiRequest({
     url: 'v2/references/data_dictionary/'
 });
+
+export const areDefCodesDisabled = (accountDownloadSubmissionTypeSelections) => (
+    accountDownloadSubmissionTypeSelections?.length === 1 &&
+    accountDownloadSubmissionTypeSelections?.includes('accountBalances')
+);
```

Comment:
@MaxwellKendall 
This works, but feels fragile to me. Is this one section of this one page the only time we foresee using this disabled logic? If so, why should it be in a separate helper library?

Reply:
It's a helper so we can test it.